tools/cert-sidecar/app/ServiceStateMgmt.js

```diff
@@ -0,0 +1,67 @@
+const {
+  ServiceStateManager: { Manager },
+} = require('@dojot/microservice-sdk');
+
+
+/**
+ * This class has responsible for the health check and graceful shutdown
+ */
+class ServiceStateMgmt {
+  /**
+   *
+   */
+  constructor() {
+    this.servername = 'certificates-sidecar';
+    this.manager = new Manager([this.servername]);
+  }
+
+  /**
+   * Adds a new health checker function.
+   *
+   * @param {(Function, Function) => void} func function to be registered as a health checker. It
+   * should receive two functions as parameters: `signalReady` and `signalNotReady`. Use them to
+   * submit the state of your service.
+   * @param {number} interval period of time (in ms) to execute the health checker.
+   *
+   */
+  addHealthChecker(func, interval) {
+    setInterval(
```

Comment:
The intervals are not being cleared when shutting down. We could leave this by now, but add this functionality to the Manager class in the SDK to properly handle them.